.github/workflows/ci.yml

```diff
@@ -131,7 +131,7 @@ jobs:
         uses: aws-actions/amazon-ecr-login@v1
       - name: Extract metadata for the Docker image
         id: meta
-        uses: descope/common/.github/actions/docker-metadata@main
+        uses: descope/.github/.github/actions/docker/metadata@feat/metadata-action
```

Comment:
Not main?
```suggestion
        uses: descope/.github/.github/actions/docker/metadata@main
```

Reply:
@omercnet Please approve this so i can modify the reference:
https://github.com/descope/.github/pull/71